pkgs/development/misc/haskell/ap-normalize/default.nix

```diff
@@ -0,0 +1,17 @@
+{ mkDerivation, base, fetchgit, inspection-testing, stdenv
+, transformers
+}:
+mkDerivation {
+  pname = "ap-normalize";
+  version = "0.1.0.1";
+  src = fetchgit {
+    url = "https://gitlab.com/lysxia/ap-normalize.git";
+    sha256 = "0xfr00dy5qiisiwkhbmmisbv3zmfak5i8fx3b9ymxswxwvvbhjnc";
+    rev = "ed521b63f02f1fe7d4e4ec5f8c5de2d4f76ab017";
+    fetchSubmodules = true;
+  };
+  libraryHaskellDepends = [ base ];
+  testHaskellDepends = [ base inspection-testing transformers ];
+  description = "Self-normalizing applicative expressions";
+  license = stdenv.lib.licenses.mit;
```

Comment:
Try upgrading your version of `cabal2nix`, it shouldn't generate `stdenv.lib` attributes anymore (which has been deprecated).

Reply:
I replaced stdenv with just lib. The warning disappeared.